webui/react/src/hooks/useResize.ts

```diff
@@ -19,7 +19,7 @@ export const DEFAULT_RESIZE_THROTTLE_TIME = 500;
 const useResize = (ref?: RefObject<HTMLElement>): ResizeInfo => {
   const [resizeInfo, setResizeInfo] = useState<ResizeInfo>(defaultResizeInfo);
 
-  useEffect(() => {
+  useLayoutEffect(() => {
```

Comment:
I think it is preferred for all uses of `useResize`.

`useLayoutEffect` essentially will allow the DOM measurements and calculations to run as early as possible, but after all the DOM measurements have been performed. In [React's description](https://react.dev/reference/react/useLayoutEffect):
```
Call useLayoutEffect perform the layout measurements before the browser repaints the screen
```
And because this hook really just needs the layout measurements to settle it doesn't have to wait for the dust to settle and wait for `useEffect` to get triggered. So the net effect is that things that depend on this hook will render the updated/measured view faster.